pkg/recipes/terraform/execute.go

```diff
@@ -218,6 +224,19 @@ func (e executor) setEnvironmentVariables(ctx context.Context, tf *tfexec.Terraf
 	return nil
 }
 
+// splitEnvVar splits a slice of environment variables into a map of keys and values.
+func splitEnvVar(env []string) map[string]string {
```

Comment:
```suggestion
func splitEnvVar(existingEnvVars []string) map[string]string {
```

Reply:
I updated it to be called envVars instead since 'existing' is a condition the calling function is using it for. pls take another look.